retrieverdash/retrieverdash/settings/common.py

```diff
@@ -139,3 +140,7 @@
             f.write(SECRET_KEY)
     except IOError:
         raise Exception('Could not open %s for writing!' % SECRET_FILE)
+
+CRONJOBS = [
+    ('*/1 * * * *', 'dashboard_script.dashboard_script.run')
+]
```

Comment:
Can me add a cron for the tests.

Reply:
Yes we can. Why do we need that though?

Reply:
Unlike the real scripts that will run one a week, we need to make tests that run every day to make sure our platform runs well.

Reply:
Okay. So do we need to keep a log file to know whether the test ran successfully or not because with the cron job we cannot see it running.

Reply:
Okey, let us skip that for now.